### ISSUE TITLE
fix north carolina statewide

### ISSUE THREAD

**Issue by justinelliotmeyers** [openaddresses/openaddresses#4868]
broken:
http://results.openaddresses.io/sources/us/nc/statewide

new:
https://services.nconemap.gov/secure/rest/services/NC1Map_Parcels/FeatureServer/0

**Comment by iandees** [comment]
This is basically the same source as the one we've got now. The current one is failing because it's taking too long to download. It might be better to grab counties individually.

**Comment by justinelliotmeyers** [comment]
I'll email and ask if there is a better way to download. I'd hate to lose a statewide source.

**Comment by justinelliotmeyers** [comment]
[nc.txt](https://github.com/openaddresses/openaddresses/files/4397083/nc.txt)

grabbed these this morning - think this is what you meant by counties individually. I did message them and ask for an additional way to download, but think this will work?  These links are for parcel polygons and centroids.

**Comment by MiniCodeMonkey** [comment]
I think I might have found a solution to this problem. Not sure if this is a new addition to NC One Map, but I just noticed that they publish statewide data as a single geodatabase file as well. See PR here: https://github.com/openaddresses/openaddresses/pull/4962

**Comment by justinelliotmeyers** [comment]
@MiniCodeMonkey so you can just point to a zipped fgdb or a fgdb somewhere and it works? IE https://www.nconemap.gov/pages/parcels then the https://dit-cgia-gis-data.s3.amazonaws.com/NCOM-data/parcels/NC_Parcels_fgdb.zip or where are you grabbing a new source from?  Thanks!!

**Comment by MiniCodeMonkey** [comment]
Yes, that's correct! That's the link from the page that I used. See this for the exact changes: https://github.com/openaddresses/openaddresses/pull/4962/files
